### ISSUE TITLE
Possibility to define env variables

### ISSUE THREAD

**Issue by andreasvirkus** [stefanjudis/forrest#120]
I'd _love_ the possibility to define env varibales per npm script (so my `npm run dev` and `npm run prod` could have different vars passed to them in Forrest).

**Comment by stefanjudis** [comment]
Heyo. Thanks for the request. :)

Hmm... Not sure if I'm a big fan of that. It feels a bit out of scope for Forrest.

How's your "normal" CLI flow? You've got different sessions for different npm scripts?

**Comment by andreasvirkus** [comment]
Currently I've got an `.env` file that I source straight in my terminal and then run npm scripts from there or have them set up in JetBrain's IDE run configuration. Lately I've moved away though from JetBrains editors (as have plenty of other Node.js peeps) to either Sublime, Atom or VSCode and their run configs with env variables seem a bit cluttered to me. So I was just wondering if forrest could be a nice sub for that (since the `.env` file approach is sorta project-wide and not script-specific).
